app/models/foreman_tasks/task/summarizer.rb

```diff
@@ -47,16 +47,17 @@ def update(field, counts)
       end
     end
 
-    def initialize(recent_timeframe = 24)
+    def initialize(scope, recent_timeframe = 24)
```

Comment:
Possible fix for the dashboard widgets issue might making `Task.authorized` as the default for the `scope` argument